syntax = "proto3";

message ApControlState{
	float yaw_bias = 1;
	float rudmidlr = 2;
	float rudmidrl = 3;
	repeated float tbl_stable_rpm = 4;
	repeated float tbl_stable_nrpm = 5;
	float x_gps = 6;
	float y_gps = 7;
	float z_gps = 8;
}